stable/jenkins/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: jenkins
 home: https://jenkins.io/
-version: 0.37.2
+version: 0.37.3
```

Comment:
lets bump this to 0.38.0

Reply:
@maorfr I have changed the version to 0.40.0 as 0.39.0 is already released